src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/AbstractFormStore.js

```diff
@@ -154,9 +153,6 @@ function collectTagPaths(
 
 export default class AbstractFormStore
 {
-    +data: Object;
```

Comment:
This is mostly the most critical change that this will not longer work with babel 7 as it tries to set it twice in the base and in the parent class. So at runtime an error accurs you can set a value to a readonly attribute. atleast our tests did also fail.